pkg/virt-handler/vm.go

```diff
@@ -383,6 +386,21 @@ func domainMigrated(domain *api.Domain) bool {
 	return false
 }
 
+func getPodInterfacefromFileCache(vmi *v1.VirtualMachineInstance, ifaceName string) (*network.PodCacheInterface, error) {
+	content, err := ioutil.ReadFile(fmt.Sprintf(virtutil.VMIInterfacepath, vmi.ObjectMeta.UID, ifaceName))
```

Comment:
once this file exists on a node, it won't change. Can we make a cache so the file is only read once.

something like...

```
// key is the file path, value is the contents.
// if key exists, then don't read directly from file.
map[string]*network.PodCacheInterface
```